.gitignore

```diff
@@ -0,0 +1,4 @@
+venv/
+__pycache__/
+config.yaml
+get
```

Comment:
What's this for?

Reply:
i created a get file for my other function but didn't want to push it yet

Reply:
Then you probably wanna move this file out from the project since it's not project related.